modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/replacement/DelayedDirtyPageWrite.java

```diff
@@ -44,13 +46,10 @@ public class DelayedDirtyPageWrite implements WriteDirtyPage {
     /** Replacing pages tracker, used to register & unregister pages being written. */
     private final DelayedPageReplacementTracker tracker;
 
-    /** Full page id to be written on {@link #finishReplacement} or null if nothing to write. */
+    /** Full page id to be written on {@link #finishReplacement} or {@code null} if nothing to write. */
     private @Nullable FullPageId fullPageId;
 
-    /** Byte buffer with page data to be written on {@link #finishReplacement} or null if nothing to write. */
-    private @Nullable ByteBuffer byteBuf;
-
-    /** Partition update tag to be used in{@link #finishReplacement} or null if nothing to write. */
+    /** Partition update tag to be used in{@link #finishReplacement} or {@code null} if nothing to write. */
```

Comment:
PersistentPageMemory -- partition update tag?

Reply:
fix it